src/main/java/com/signavio/examples/dmn/testsuite/TestSuiteReader.java

```diff
@@ -16,13 +16,11 @@
 	private static final Gson GSON = new GsonBuilder().setPrettyPrinting().create();
 	
 	
-	// TODO: TestSuite.class is part of the bdm-test-suite-api, clarify if Signavio is publishing it
 	public TestSuite parse(InputStream testSuite) {
 		try {
 			JsonReader reader = createTestSuiteReader(testSuite);
 			JsonObject testSuiteJson = GSON.fromJson(reader, JsonObject.class);
```

Comment:
So now we have Jackson and GSON in here? :thinking:

Reply:
This is legacy and i will remove it, lets stick to Jackson because this is already available.